versions.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# Sources for a base cloud os images
+cloudimg:
+  ubuntu:
+    focal:
+      amd64:
+        url: https://cloud-images.ubuntu.com/releases/focal/release-20230107/ubuntu-20.04-server-cloudimg-amd64.img
+        checksum: 3895e38566e5c2c019f5c6f825ab7570ee34dac6b9142fab0c7e5a78084c4280
+      s390x:
+        url: https://cloud-images.ubuntu.com/releases/focal/release-20230107/ubuntu-20.04-server-cloudimg-s390x.img
+        checksum: 24673aa86785573d3a92e15166ff81beff88cbb0abc01938f156eb1332e87cd3
+  centos:
+    stream-8:
+      amd64:
+        url: https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2
+        checksum: b80ca9ccad8715aa1aeee582906096d3190b91cebe05fcb3ac7cb197fd68a7fe
+  redhat:
+    rhel8:
+      amd64:
+        url:
+        checksum:
+# Fixed version references
+version:
```

Comment:
how about using `tools` or `build-tools` instead of `version` for golang, rust, protoc, packer, docker/buildx ?

Reply:
@jtumber-ibm looks like the earlier change got reverted